src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -629,7 +629,7 @@
     <Reference Include="System.Net.NameResolution" />
     <Reference Include="System.Net.NetworkInformation" />
     <Reference Include="System.Net.Primitives" />
-    <Reference Include="System.Net.Quic" />
+    <ProjectReference Include="..\..\System.Net.Quic\ref\System.Net.Quic.csproj" />
```

Comment:
nit: please move this up to the other ProjectReference as we usually sort alphabetically. Also use the `$(LibrariesProjectRoot)` property instead of `..\..\`.

Reply:
Fixed.